dependencies.yaml

```diff
@@ -334,6 +335,12 @@ dependencies:
       - output_types: [conda]
         packages:
           - curl
+  # TODO: Remove geopandas.dataset usage in cuspatial_api_examples.ipynb
+  test_notebooks:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - geopandas<1
```

Comment:
@mroeschke Can you open an issue documenting the `geopandas<1` pinning and whatever work items need to be done there?

I am going to merge this once CI passes to unblock #1402 since it is blocking other RAPIDS-CCCL work, just want to be sure that this is visible and has a corresponding issue.

Reply:
Sure thing opened https://github.com/rapidsai/cuspatial/issues/1403